### ISSUE TITLE
"brew install openssl" doesn't install 10.5.8 G4

### ISSUE THREAD

**Issue by cbradburne** [mistydemeo/tigerbrew#797]
Hi,
Any recommendations on how to install openssl using Tigerbrew on G4 10.5.8?
All it does is point me to about 9 different GitHub issues which don't help me install openssl.
Sorry if it's a simple fix that I'm missing.

Thanks in advance.

```
G4-Cube:~ cube$ brew install openssl
==> Downloading https://www.openssl.org/source/old/1.0.2/openssl-1.0.2j.tar.gz
Already downloaded: /Users/cube/Library/Caches/Homebrew/openssl-1.0.2j.tar.gz
==> Downloading https://trac.macports.org/export/144472/trunk/dports/devel/openssl/files/x86_64-asm-on-i386.patch
Already downloaded: /Users/cube/Library/Caches/Homebrew/openssl--patch-98ffb308aa04c14db9c21769f1c5ff09d63eb85ce9afdf002598823c45edef6d.patch
==> Patching
==> Applying x86_64-asm-on-i386.patch
patching file crypto/perlasm/x86_64-xlate.pl
Hunk #1 succeeded at 261 with fuzz 2 (offset 66 lines).
==> perl ./Configure --prefix=/usr/local/Cellar/openssl/1.0.2j --openssldir=/usr/local/etc/openssl no-ssl2 zlib-dynamic shared enable-cms darwin-ppc-cc
Last 15 lines from /Users/cube/Library/Logs/Homebrew/openssl/01.perl:
2022-11-18 20:51:06 +0000

perl
./Configure
--prefix=/usr/local/Cellar/openssl/1.0.2j
--openssldir=/usr/local/etc/openssl
no-ssl2
zlib-dynamic
shared
enable-cms
darwin-ppc-cc


READ THIS: https://github.com/mistydemeo/tigerbrew/wiki/troubleshooting

These open issues may also help:
openssl source url is dead https://github.com/mistydemeo/tigerbrew/issues/670
Updated openssl to 1.1.1i https://github.com/mistydemeo/tigerbrew/pull/722
error with openssl and a workaround https://github.com/mistydemeo/tigerbrew/issues/117
OpenSSL --universal build fail on Leopard  https://github.com/mistydemeo/tigerbrew/issues/382
Update OpenSSL and enable universal build on Tiger https://github.com/mistydemeo/tigerbrew/pull/488
OpenSSL does not build with --universal option on Mac OS 10.4 https://github.com/mistydemeo/tigerbrew/issues/463
OpenSSL 1.0.2j failed to build on 10.5.8 on g5_64 https://github.com/mistydemeo/tigerbrew/issues/462
tor.rb failed to build on  10.5.8(ppc/G4) dl-links dead, new tor requires openssl@1.1 https://github.com/mistydemeo/tigerbrew/issues/675
aria2: Updated to 1.36.0. Added OS X 10.4.x build support. Fixed building on 10.5.x as well as issue #394. Switched SSL library from GnuTLS to OpenSSL due to some process hanging issues with the former. Added `expat` for XML support. https://github.com/mistydemeo/tigerbrew/pull/768
G4-Cube:~ cube$ 
```

**Comment by mistydemeo** [comment]
Can you please attach all the logs from `~/Library/Logs/Homebrew/openssl`? You can upload those to gist.github.com. Thanks!

**Comment by cbradburne** [comment]
Thanks for getting back to me so quickly.

I've never created a gist before, I hope this is right.
There was only one file in that location, here's the gist of it:

[gist link](https://gist.github.com/cbradburne/2790a392c87ecb88c6a30d86cd969df4)

**Comment by mistydemeo** [comment]
Hm, that's funny. What happens if you type `which perl` in your terminal?

**Comment by cbradburne** [comment]
Thanks for prompting me to check the perl install.
I was attempting to follow this install for Shairport [LINK](https://medium.com/@bram.beirens/install-shairport-airplay-server-on-a-powerpc-g4-mac-running-leopard-10-5-8-in-2022-e0205eb0e9d5).
However, I was getting a -Werror=declaration-after-statement error due to perl 5.22.0 not including the fix for that error. After trying many things, I installed a later version of perl via Macport. I was then able to "make" shairport, but running the shairport.pl resulted in the openssl error.

Since, I have uninstalled Macports and reinstalled "brew install perl" and things seem to be working now.
Ultimately I should've come here first when I had the initial -werror, as now I can't create that same error and that doesn't help anyone.

Thank you for your help.
Colin
